src/Rotations.jl

```diff
@@ -1,5 +1,5 @@
 # we generate code in this module, so precompile where possible
-#__precompile__(true)
+__precompile__(true)
```

Comment:
IIRC this was commented out due to a problem with Quaternions.  Unless we can get that upgraded, we might be stuck here too?  (Seems like Quaternions has no maintainer at the moment.)